### DISCUSSION TITLE
[New feature] PiP & Auto next video in profile

### DISCUSSION THREAD

**Discussion by thaibm92** [dayanch96/BHTikTok-Plus#23] [General]
I want to let you know that it's great of you to have developed this Plus version.
If you can add PiP and Auto next video feature in profile, it will be even better. I and other users in Vietnam thank you very much.

**Comment by dayanch96** [comment]
I did not added autoskip and new pure mode in profiles since it profiles - not feed. its rare when you keep scrolling profile videos. but ill take a look

And thanks for using, translating bhtiktokplus, i appreciate it
